src/widgets/modals/ByProgramModal.js

```diff
@@ -266,7 +296,7 @@ export const ByProgramModal = ({ settings, database, transactionType, onConfirm
 
   return (
     <FlexColumn flex={1} alignItems="center">
-      <ProgramToggleBar />
+      {transactionType !== 'customerRequisition' ? <ProgramToggleBar /> : null}
```

Comment:
Cheap hack hiding the toggle bar (Creating general requisitions not part of the spec)